keps/sig-storage/4958-csi-sidecars-all-in-one/kep.yaml

```diff
@@ -0,0 +1,41 @@
+title: CSI Sidecars All in one
+kep-number: 4958
+authors:
+  - "@mowangdk"
+  - "@mauriciopoppe"
+  - "@ConnorJC3"
+owning-sig: sig-storage
+participating-sigs:
+status: implementable
+creation-date: 2024-11-10
+reviewers:
+  - "@xing-yang"
+  - "@jsafrane"
+approvers:
+  - "@xing-yang"
+  - "@jsafrane"
+
+see-also:
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.34"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.34"
```

Comment:
1.34?

Reply:
I believe this still stands.

Reply:
Same for the `latest-mileston` entry above.